src/zsock.c

```diff
@@ -2376,6 +2376,7 @@ zsock_test (bool verbose)
     zstr_free (&addr);
     zstr_free (&message);
 
+#ifndef __APPLE__
```

Comment:
this smells a bit - any idea why multicast v4 wouldn't work on osx?

Reply:
Of course it smells, but right now I can't run the tests unless this is commented out becauseI don't know how to re-configure the my Mac's networking to make it work. I'd be fine to keep this as a git stash to be able to work, but then everyone else will run into the same problem and will need to figure this out, costing him valuable time. Maybe we could create a bug ticket for someone t figure it out?

Reply:
I think multicast is just not configured on Macs, although they're capable. Only found a very old post: https://blogs.agilefaqs.com/2009/11/08/enabling-multicast-on-your-macos-unix/. Did not want to go down this rabbit hole.